apps/mobile/src/libs/components/background-wrapper/libs/components/planet/planet.tsx

```diff
@@ -0,0 +1,36 @@
+import { type StyleProp, type ViewStyle } from "react-native";
+import LinearGradient from "react-native-linear-gradient";
```

Comment:
The same for these types and export LinearGradient from ~libs/components/components